src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1146,10 +1147,12 @@ protected override void GetItem(string path)
                         }
                     }
                 }
+#endif
 
                 FileSystemInfo result = GetFileSystemItem(path, ref isContainer, false);
                 if (result != null)
                 {
+#if !UNIX
```

Comment:
I think the other case I was fine with a bit of duplication, but prefer to leave this one as-is.